packages/core/src/App/Containers/AccountSignupModal/account-signup-modal.jsx

```diff
@@ -49,6 +50,16 @@ const AccountSignup = ({ enableApp, is_mobile, isModalVisible, clients_country,
             }
             setIsLoading(false);
         });
+
+        Analytics.trackEvent('ce_virtual_signup_form', {
+            action: 'signup_confirmed',
+            form_name: is_mobile ? 'virtual_signup_web_mobile_default' : 'virtual_signup_web_desktop_default',
+        });
```

Comment:
since it's repeatable action
and what about `signup_provider` and `form_name: string | 'virtual_signup_web_mobile_default' | 'virtual_signup_web_desktop_default' | 'virtual_signup_mobile_app_default'`?
```suggestion
const analyticsData: Parameters<typeof Analytics.trackEvent>[1] = {
    form_source: is_mobile ? 'virtual_signup_web_mobile_default' : 'virtual_signup_web_desktop_default',
}

        Analytics.trackEvent('ce_virtual_signup_form', {
            action: 'signup_confirmed',
            ...analyticsData
        });
```